src/components/Button/index.js

```diff
@@ -1,16 +1,15 @@
 import React from 'react';
+
 import { Button as MuiButton, Typography } from '@material-ui/core';
 
 export default function Button(props) {
   // One option to get theme and supply it to styled compoenents
   return (
     <MuiButton color="primary" variant="contained">
-      <Typography variant="button">
-        Order Food
-      </Typography>
+      <Typography variant="button">Order Food</Typography>
```

Comment:
I think if it is a common component we must use props to display the title of the button ..